.travis.yml

```diff
@@ -65,7 +73,7 @@ jobs:
       php: "7.0"
       dist: trusty
       env:
-        - SERVER_DISTRO=ubuntu1404
+        - SERVER_DISTRO=enterprise-ubuntu1404
```

Comment:
We can't run FLE tests, but with the changed URL we won't be able to install the Community Edition from that URL. Instead of customising the download URL as well I decided to switch this build to an enterprise package instead.